sdk/servicebus/azure-servicebus/azure/servicebus/exceptions.py

```diff
@@ -491,6 +493,7 @@ class AutoLockRenewTimeout(ServiceBusError):
     AMQPErrorCodes.UnauthorizedAccess: ServiceBusAuthorizationError,
     AMQPErrorCodes.NotImplemented: ServiceBusError,
     AMQPErrorCodes.NotAllowed: ServiceBusError,
+    AMQPErrorCodes.LinkDetachForced: ServiceBusConnectionError,
```

Comment:
What kind of error was this raised as before?

Reply:
it used to be the general `ServiceBusError` which would not be retried.

but after contemplating over it, I think this should be categorized into `ServiceBusConnectionError` and be retried.